### ISSUE TITLE
Email domain input missing aria-describedby attribute

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-cas-sso#226]
The Email Domain input on the Central Authentication Service (CAS) page is not associated with input description via an aria-describedby attribute.
